LayoutTests/http/wpt/webauthn/public-key-credential-create-with-invalid-parameters.https.html

```diff
@@ -43,6 +43,8 @@
         [{ name: rp.name, id: Symbol() }, user, challenge, [pubKeyCredParam]],
         // wrong user attribute type
         [rp, { name: Symbol(), id: user.id, displayName: user.displayName}, challenge, [pubKeyCredParam]],
+        [rp, { name: user.name, id: asciiToUint8Array(""), displayName: user.displayName}, challenge, [pubKeyCredParam]],
+        [rp, { name: user.name, id: asciiToUint8Array("12345678123456781234567812345678123456781234567812345678123456781"), displayName: user.displayName}, challenge, [pubKeyCredParam]],
```

Comment:
It seems like we have a test case for a single-character id. Do we also have one for a 64-byte ID?

Clearly you also test the < 1 and > 64 cases in this change.

Reply:
Good idea, added test cases.